db/schema.rb

```diff
@@ -845,6 +845,14 @@
     t.index ["uuid"], name: "index_user_notifications_on_uuid", unique: true
   end
 
+  create_table "user_preferences", charset: "utf8mb4", collation: "utf8mb4_unicode_ci", force: :cascade do |t|
+    t.bigint "user_id"
+    t.boolean "auto_update_exercises", default: true, null: false
```

Comment:
This defaulting to `true` might be something we want to mention in the newsletter.